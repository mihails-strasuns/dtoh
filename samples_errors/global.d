module global;

// must be __gshared
extern(C) int var;
