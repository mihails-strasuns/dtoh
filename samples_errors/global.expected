samples_errors/global.d(4) [int]
	Missing __gshared on extern(C) global variable
